README.md

```diff
@@ -0,0 +1,30 @@
+
+![](./images/MB.png)
+
+# System Overview.
+
+The Motherboard has seven identical slots for ![](https://latex.codecogs.com/gif.latex?35mm%20%5Ctimes%2040mm) modules which
```

Comment:
i think you could put the math inside the description: `![35mm \times 40mm](https...)`